exporters/otlp/src/recordable.cc

```diff
@@ -213,6 +324,8 @@ void Recordable::SetSpanKind(opentelemetry::trace::SpanKind span_kind) noexcept
   span_.set_kind(proto_span_kind);
 }
 
+void SetResource(const opentelemetry::sdk::resource::Resource &resource) noexcept {}
```

Comment:
Is this global function needed?

Reply:
Thanks for noticing it. No idea why I added it, removed it now.